.buildkite/browser-pipeline.yml

```diff
@@ -73,10 +70,18 @@ steps:
             - with: { browser: safari, version: 18 }
 
       # BrowserStack
-      - label: ":browserstack: :{{ matrix.browser }}: {{ matrix.version }} Browser tests${EXTRA_STEP_LABEL}"
-        depends_on: "browser-maze-runner-bs-${USE_CDN_BUILD}"
+      - label: ":browserstack: :{{ matrix.browser }}: {{ matrix.version }} Browser tests - ${BUILD_MODE}"
+        depends_on: "build-browser-fixtures-${BUILD_MODE}"
         timeout_in_minutes: 30
         plugins:
+          artifacts#v1.9.4:
+            download:
+              - "fixture-packages-${BUILD_MODE}.zip"
+              - from: "test/browser/features/fixtures/.bugsnag-browser-version-${BUILD_MODE}"
+                to: "test/browser/features/fixtures/.bugsnag-browser-version"
+            upload: "./test/browser/maze_output/failed/**/*"
+          improbable-eng/metahook:
+            pre-command: "unzip -o fixture-packages-${BUILD_MODE}.zip"
```

Comment:
This is a slightly annoying dance we have to do, due to the fact that the artifacts plugin has poor support for compressed artifacts (the `compressed` attribute is shard between the upload and download).